modules/EvseV2G/iso_server.cpp

```diff
@@ -178,12 +178,18 @@ static bool load_contract_root_cert(mbedtls_x509_crt* contract_root_crt, const c
                                     const char* MO_file_path) {
     int rv = 0;
 
-    if (((rv = mbedtls_x509_crt_parse_file(contract_root_crt, MO_file_path)) != 0) &&
-        ((rv = mbedtls_x509_crt_parse_file(contract_root_crt, V2G_file_path)) != 0)) {
+    if ((rv = mbedtls_x509_crt_parse_file(contract_root_crt, MO_file_path)) != 0) {
         char strerr[256];
         mbedtls_strerror(rv, strerr, 256);
-        dlog(DLOG_LEVEL_ERROR, "Unable to parse V2G (%s) or MO (%s) root certificate  (err: -0x%04x - %s)",
-             V2G_file_path, MO_file_path, -rv, strerr);
+        dlog(DLOG_LEVEL_ERROR, "Unable to parse MO (%s) root certificate. (err: -0x%04x - %s)",
+             MO_file_path, -rv, strerr);
+        dlog(DLOG_LEVEL_INFO, "Attempting to parse V2G root certificate..");
+
+        if ((rv = mbedtls_x509_crt_parse_file(contract_root_crt, V2G_file_path)) != 0) {
+            mbedtls_strerror(rv, strerr, 256);
```

Comment:
Personally i would prefer to use sizeof(strerr)

Reply:
Done..